.github/workflows/ci.yml

```
@@ -16,7 +27,10 @@ permissions:
 jobs:
   fmt:
     name: Check formatting
-    runs-on: blacksmith-4vcpu-ubuntu-2404
+    # `cargo fmt --check` parses and prints. It is setup- and I/O-bound and
+    # never saturates one core, let alone four. Smallest ARM runner:
+    # $0.0025/min against the $0.008 it costs today.
+    runs-on: blacksmith-2vcpu-ubuntu-2404-arm
```

actionlint (1.7.12)

[error] label "blacksmith-2vcpu-ubuntu-2404-arm" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-intel", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)